runtime/tools/python/setup.py

```diff
@@ -18,6 +18,10 @@
     "SOURCE_ROOT",
     os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "..", ".."),
 )
+ttmlir_build_dir = os.environ.get(
```

Comment:
Maybe default to `{src_dir}/build` unless this var is set?

Reply:
Thanks; updated to use `{src_dir}/build` as default if env variable is not set.